autohooks/cli/plugins.py

```diff
@@ -73,11 +73,11 @@ def add_plugins(term: Terminal, args: Namespace) -> None:
 
     if config.has_autohooks_config():
         settings = config.settings
-        existing_plugins = set(settings.pre_commit)
+        existing_plugins = set(settings.pre_commit)  # type: ignore
```

Comment:
in multiple places throughout the code i placed some "# type: ignore" to ignore mypy errors that occur because "settings" might not be definied. But since we checked earlier if config has an autohooks config (line 74), this should be not an error at all. So i think these errors can simply be ignored. If you believe there might be an easier solution please share it with me.

Reply:
(or a cleaner solution)

Reply:
Should be find to just add an ignore statement here.